l1-contracts/src.ts/hyperchain-upgrade.ts

```diff
@@ -181,9 +233,70 @@ async function integrateEraIntoBridgehubAndUpgradeL2SystemContract(deployer: Dep
 }
 
 async function upgradeL2Bridge(deployer: Deployer) {
-  // upgrade L2 bridge contract, we do this directly via the L2
-  // set initializeChainGovernance in L1SharedBridge
-  deployer;
+  // todo deploy l2 bridge here
```

Comment:
does it have to be solved within this pr?

Reply:
No, I will probably do it for localhost testing though